src/modes/linear_world.cpp

```diff
@@ -262,6 +268,45 @@ void LinearWorld::updateGraphics(float dt)
 
 }   // updateGraphics
 
+// ----------------------------------------------------------------------------
+/** This calculate the time difference between the second kart in the
+ *  race and the first kart in the race (who must be a ghost)
+ */
+void LinearWorld::updateLiveDifference()
```

Comment:
Not a big fan of adding so much ghost-specific stuff to linear world, isn't it possible to move that to some ghost class?

Reply:
+1

Reply:
This doesn't require to be tied to ghosts (i.e., it could potentially be used elsewhere in the future), so I'm not a fan either of moving it to a ghost-specific class.

Maybe it could use its own class, somewhere in the kart or race directory ? (not sure about the most appropriate directory) It would be a relatively small one. If that way is chosen, I'd rewrite it to be able to work with more than 2 karts.